conversion.md

```diff
@@ -46,17 +46,21 @@ These fields all affect the `annotations` of the runtime configuration, and are
 | ------------------- | --------------- | ----- |
 | `os`                | `annotations`   | 1,2   |
 | `architecture`      | `annotations`   | 1,3   |
-| `author`            | `annotations`   | 1,4   |
-| `created`           | `annotations`   | 1,5   |
+| `variant`           | `annotations`   | 1,4   |
+| `os.version`        | `annotations`   | 1,5   |
+| `os.features`       | `annotations`   | 1,6   |
```

Comment:
There isn't a corresponding footnote for either annotation mapping.

Reply:
🤦 Fixed.